### PR TITLE
strategies: init at 0.2.3

### PR DESCRIPTION
**Pull request by suhr**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Assured whether relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/development/python-modules/strategies/default.nix [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "02i4ydrs9k61p8iv2vl2akks8p9gc88rw8031wlwb1zqsyjmb328";
+  };
+
+  propagatedBuildInputs = [ multipledispatch toolz ];
+  checkInputs = [ pytest ];
+
+  meta = {
```
Comment (mmahut) [inline pkgs/development/python-modules/strategies/default.nix]: Missing ``with stdenv.lib;``.

**Review by mmahut — CHANGES_REQUESTED** [review]

File: pkgs/development/python-modules/strategies/default.nix (hunk 1 above)
Comment (alexarice) [reply pkgs/development/python-modules/strategies/default.nix]: Think that is fine here

Comment (suhr) [reply pkgs/development/python-modules/strategies/default.nix]: Sympy doesn't have it too: https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/python-modules/sympy/default.nix#L37

Comment (alexarice) [reply pkgs/development/python-modules/strategies/default.nix]: Does seem unusual to have lib as an argument at the top just for meta though

Comment (Ma27) [reply pkgs/development/python-modules/strategies/default.nix]: It's just a syntactical difference and a lot of packages skip the `with stdenv.lib` part. Resolving this for now as it's just a question of style.

**Comment by Ma27** [comment]
Built and tested locally :+1:

**Comment by Ma27** [comment]
@suhr thanks!

**Pull request merged**